extensions_admin/pulp_docker/extensions/admin/upload.py

```diff
@@ -30,16 +31,33 @@ def __init__(self, context):
 
     def determine_type_id(self, filename, **kwargs):
         """
-        Determine whether it is a V1 or V2 image
+        Determine type id of the upload file by file type.
+
+        json -> Manifest List
+        tarfile -> Image or Manifest
```

Comment:
v1 Image or v2 Image Manifest. Both of them are images, just different api version.